common/src/main/scala/com/stratio/crossdata/common/serializers/CommandSerializer.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stratio.crossdata.common.serializers
+
+import com.stratio.crossdata.common.{CloseSessionCommand, Command, OpenSessionCommand, SQLCommand}
+import org.json4s._
+import CommandSerializerHelper._
+
+import scala.reflect.ClassTag
+
+private [serializers] object CommandSerializerHelper {
+
+  case class CommandWithName[T <: Command : ClassTag](command: String, details: Option[T] = None)
+
+  object CommandWithName {
+    def apply[T <: Command : ClassTag](details: Option[T]): CommandWithName[T] =
```

Comment:
`CommandSerializerHelper` is not `CommandSerializer`'s companion object therefore the farthest  I can limit the scope is `private[serializers]`